scripts/populate_conferences.py

```diff
@@ -175,6 +175,17 @@ def main():
         ],
         'public_projects': True,
     },
+    +    'TeaP2015': {
```

Comment:
Looks like you added the '+' signs from a diff. @saradbowman Can you please remove those and fix the indentation?